phpunit/fonts-library/class-wp-font-family-utils-test.php

```diff
@@ -0,0 +1,320 @@
+<?php
+/**
+ * Tests for Font Family class
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts Library
+ */
+
+/**
+ * @coversDefaultClass WP_Font_Family_Utils
+ */
+class WP_Font_Family_Utils_Test extends WP_UnitTestCase {
+
+	/**
+	 * @covers ::has_font_mime_type
+	 *
+	 * @dataProvider data_has_font_mime_type_fixtures
+	 *
+	 * @param string $font_file Font file path.
+	 * @param bool   $expected  Expected result.
+	 */
+	public function test_has_font_mime_type( $font_file, $expected ) {
+		$this->assertSame( $expected, WP_Font_Family_Utils::has_font_mime_type( $font_file ) );
+	}
+
+	/**
+	 * Data provider.
+	 *
+	 * @return array[]
+	 */
+	public function data_has_font_mime_type_fixtures() {
+		return array(
+			'ttf'   => array(
+				'font_file' => '/temp/piazzolla_400_italic.ttf',
+				'expected'  => true,
+			),
+			'otf'   => array(
+				'font_file' => '/temp/piazzolla_400_italic.otf',
+				'expected'  => true,
+			),
+			'woff'  => array(
+				'font_file' => '/temp/piazzolla_400_italic.woff',
+				'expected'  => true,
+			),
+			'woff2' => array(
+				'font_file' => '/temp/piazzolla_400_italic.woff2',
+				'expected'  => true,
+			),
+			'exe'   => array(
+				'font_file' => '/temp/piazzolla_400_italic.exe',
+				'expected'  => false,
+			),
+			'php'   => array(
+				'font_file' => '/temp/piazzolla_400_italic.php',
+				'expected'  => false,
+			),
+		);
+	}
+
+	/**
+	 * @covers ::get_filename_from_font_face
+	 *
+	 * @dataProvider data_get_filename_from_font_face_fixtures
+	 *
+	 * @param string $slug               Font slug.
+	 * @param array  $font_face          Font face data in theme.json format.
+	 * @param string $suffix Suffix added to the resulting filename. Default empty string.
+	 * @param string $expected_file_name Expected file name.
```

Comment:
```suggestion
	 * @param string $slug               Font slug.
	 * @param array  $font_face          Font face data in theme.json format.
	 * @param string $suffix             Suffix added to the resulting filename. Default empty string.
	 * @param string $expected_file_name Expected file name.
```
Minor alignment fix.